structure/0048-outreach-team.md

```diff
@@ -0,0 +1,71 @@
+# Outreach Team
+
+This document has been made according to the process outlined in [RFC
+0006][RFC6].
+
+## Summary
+
+The Outreach Team is a subteam of the Community Team in charge of outreach
+and communications within and outside the Quilt Project. This team is also
+responsible for working with the development teams to communicate the
+current status of development progress with the community and with any
+partners. In essence, it may be thought of as a "public relations
+department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, announcements, or team to-do lists.
+Historically, this has been managed by the Community Team as a whole.
+However, communications don't fall within the realm of the current Quilt
+community team, and thus this should have its own separate subteam.
+
+## Explanation
+
+The Outreach Team will be responsible for all communications outside of and
+within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts and announcements
+- managing the official QuiltMC social medias, with the exception of Discord
```

Comment:
Could change this to "social media accounts", and drop the part about Discord - this would be better IMO since it more accurately describes things and won't need updating if we add/move to other chat platforms

Reply:
Well, the initial members need to be whoever is managing the systems currently covered under the RFC. So the ones I can think of:
- You, because I think you manage the Twitter account (with i5)?
- Toffee, because he writes the blog posts (although we need more people on that, I think, with just him the updates are inconsistent)
- Me, because I do the podcast.
- Someone who's familiar enough with the toolchain to manage the GitHub project boards, maybe Oro or Glitch?

Reply:
Maybe “social media, with the exception of forums and chatrooms”?

That covers Discord, and also provides the opportunity for the community team to step in with moderating, for example, the subreddit if they feel the need to.